cpp/OcvPersonDetection/plugin-files/descriptor/descriptor.json

```diff
@@ -60,19 +60,19 @@
         },
         {
           "name": "MERGE_TRACKS",
-          "description": "When set to true, the detection response handler will merge tracks spanning segment boundaries.",
+          "description": "In the context of videos, when set to true, the detection response handler will merge tracks spanning segment boundaries.",
           "type": "BOOLEAN",
           "defaultValue": "false"
```

Comment:
The default values for the 3 track properties should be specified via a propertiesKey because they appear in mpf.properties.